webapp/channels/src/sass/components/_post.scss

```diff
@@ -70,6 +70,10 @@
     }
 }
 
+.display-none {
+    display: none;
+}
```

Comment:
we want to avoid these utility css classes

Reply:
will using inline styles be a fair solution ?. Haven't used inline styles are they were less frequently used in the repository.

```
<div
    ref={this.wrapper}
    className='textarea-wrapper'
    style={{"display": this.props.preview ? "none": "block"}}
>
```

I might abstract the style conditional in a separate place.

Reply:
have refactored as per [this](https://github.com/mattermost/mattermost/pull/25085#discussion_r1372311503) suggestion